types/assert-equal-jsx/assert-equal-jsx-tests.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {assertEqualJSX} from 'assert-equal-jsx';
```

Comment:
What exactly is wrong about this pattern by the way? Isn't the intended pattern to be to declare types for items that don't already have declarations?

I thought this is exactly what is supposed to happen, let me know if I'm wrong.